### PR TITLE
Updated to add repos for devplatform for depenapanda

### PR DESCRIPTION
**Pull request by andyloughran**
Adds the dev-platform team repos for PR monitoring.

### CODE REVIEW

File: config/alphagov.yml [hunk 1]
```diff
@@ -184,6 +184,19 @@ dev-platform-team:
...
+    - di-devplatform-utils
+    - di-devplatform-identity-broker 
+    - di-devplatform-infra-audit-hook 
+    - di-devplatform-lambda-audit-hook 
+    - di-devplatform-upload-action
+    - di-devplatform-upload-action-ecr 
+    - di-devplatform-metriccollector
+    - di-devplatform-checkov-hook 
+    - di-devplatform-container-verifier 
+    - di-devplatform-auth-api-test
```
Comment (beca-galliano) [inline config/alphagov.yml:199]: I can't find this one on GitHub but maybe I don't have access to it 🤔

Comment (jamesmelville-gds) [reply config/alphagov.yml:199]: similar, does https://govukverify.atlassian.net/wiki/spaces/PLAT/pages/3170893873/GitHub+Repositories need this adding to it?

File: config/alphagov.yml [hunk 2]
```diff
@@ -184,6 +184,19 @@ dev-platform-team:
     - 'Remember to celebreate our successes :tada:'
     - We believe in the power of documentation.
     - We believe that no bug should occur twice, but if it does, we will be able to fix it off documentation.
```
Comment (jamesmelville-gds) [inline config/alphagov.yml]:
```suggestion
    - We believe that no bug should occur twice, but if it does, we will be able to fix it following documentation.
```

File: config/alphagov.yml [hunk 3]
```diff
@@ -184,6 +184,19 @@ dev-platform-team:
     - 'Remember to celebreate our successes :tada:'
```
Comment (jamesmelville-gds) [inline config/alphagov.yml]:
```suggestion
    - 'Remember to celebrate our successes :tada:'
```

**Review by jamesmelville-gds — COMMENTED** [review]
The whole dev platform section ought to be moved to L74 so that the sections are listed alphabetically

File: config/alphagov.yml (hunk 1 above)
Comment (jamesmelville-gds) [reply config/alphagov.yml:199]: and the dynatrace one is not here, but perhaps that needs to be added to its own section

**Review by MuriloDalRi — APPROVED** [review]

**Pull request merged**